packages/plugins/other/visitor-plugin-common/src/selection-set-to-object.ts

```diff
@@ -548,6 +548,7 @@ export class SelectionSetToObject<Config extends ParsedDocumentsConfig = ParsedD
     selectionNodes = [...selectionNodes];
     let inlineFragmentConditional = false;
     for (const selectionNode of selectionNodes) {
+      // 1. Handle Field or Directtives selection nodes
```

Comment:
```suggestion
      // 1. Handle Field or Directives selection nodes
```

Noticed a small typo 🙂